media/js/libs/ckeditor/skins/kuma/dialog.css

```diff
@@ -104,3 +104,21 @@ body .cke_dialog{visibility:visible;padding:20px;background:rgba(0,0,0,.3);font:
   background: -webkit-linear-gradient(top, #f8b8a0 0, #f8a688 100%);
   background: linear-gradient(top, #f8b8a0 0, #f8a688 100%);
 }
+
+.cke_skin_kuma #cke_dialog_contents_78 div.cke_dialog_ui_input_text, 
+.cke_skin_kuma #cke_dialog_contents_78 div.cke_dialog_ui_input_password {
+  border:none;
+}
+
+.cke_skin_kuma #cke_dialog_contents_78 input.cke_dialog_ui_input_text, 
+.cke_skin_kuma #cke_dialog_contents_78 input.cke_dialog_ui_input_password {
+  border:solid 1px #cbc8b9;
+}
```

Comment:
I notice that these changes cause the input borders to appear mis-aligned with the input headings. In other words, the left side of the border appears a few pixels off from the left side of the heading.

It looks like the bug is still corrected without these changes. Was there another reason these changes were added? If not, we could safely remove them.